Foundation/Data.swift

```diff
@@ -2103,7 +2103,7 @@ public struct Data : ReferenceConvertible, Equatable, Hashable, RandomAccessColl
     }
     
     @available(swift, introduced: 4.2)
-    @available(swift, deprecated: 5, message: "use `init(_:)` instead")
+    @available(swift, deprecated: 5, renamed: "init(_:)")
```

Comment:
cc @phausler we should have this in the overlay, too.

Reply:
I agree, this is superior for a user experience. We should definitely do it.